src/rules/functionality.ts

```diff
@@ -72,12 +74,18 @@ export default {
   'no-unsafe-finally': true,
   // Disallows unused expression statements.
   'no-unused-expression': true,
+  // Disallows unused imports, variables, functions and private class members. Similar to tsc’s –noUnusedParameters and –noUnusedLocals options, but does not interrupt code compilation.
+  'no-unused-variable': true,
   // Disallows usage of variables before their declaration.
   'no-use-before-declare': true,
   // Disallows usage of the `var` keyword.
   'no-var-keyword': true,
   // Requires expressions of type `void` to appear in statement position.
   'no-void-expression': false,
+  // Recommends to use a conditional expression instead of assigning to the same thing in each branch of an if statement. <FEEDBACK>
+  'prefer-conditional-expression': false,
```

Comment:
Ehhhhhhh.  I say leave it off.  It'll cause unnecessary flipping between if/ternary expressions.  Harder `git blame`.

[Examples](https://github.com/palantir/tslint/blob/master/test/rules/prefer-conditional-expression/test.ts.lint)

It looks like [only a single file in neutron is affected](server/middleware/error-response.ts).